src/app/data-model-provider/Provider.h

```diff
@@ -90,18 +86,17 @@ class Provider : public ProviderMetadataTree
     ///     - Validation of readability/writability (also controlled by OperationFlags::kInternal)
     ///     - Validation of timed interaction required (also controlled by OperationFlags::kInternal)
     ///
-    /// Return codes
-    ///   CHIP_IM_GLOBAL_STATUS(code):
-    ///       - error codes that are translatable to specific IM codes
-    ///       - in particular, the following codes are interesting/expected
-    ///         - `UnsupportedWrite` for attempts to write read-only data
-    ///         - `UnsupportedAccess` for ACL failures
-    ///         - `NeedsTimedInteraction` for writes that are not timed however are required to be so
-    virtual CHIP_ERROR WriteAttribute(const WriteAttributeRequest & request, AttributeValueDecoder & decoder) = 0;
+    /// Return notes:
+    ///    - The following codes are interesting/expected to be possible:
```

Comment:
There were called out before. Removed since I believe we called them out as examples for CHIP_IM_GLOBAL_ERROR but now that we have real statuses, we should not care about this as much.